test/Microsoft.IdentityModel.JsonWebTokens.Tests/Properties/AssemblyInfo.cs

```diff
@@ -5,7 +5,7 @@
 using System.Runtime.InteropServices;
 using Xunit;
 
-[assembly: CLSCompliant(true)]
+[assembly: CLSCompliant(false)]
```

Comment:
Why are these CLSCompliant attributes changing?

Reply:
We don't need to be CLSCompliant in test.

Reply:
I would just remove these lines then. If you don't have a CLSCompliant attribute, the assembly is not CLSCompliant.